tests/test_arcaflow_plugin_pcp.py

```diff
@@ -146,6 +146,46 @@ def test_functional_flat(self):
         self.assertEqual("success", output_id)
         self.assertIsInstance(output_data.pcp_output, list)
 
+    def test_functional_user_pmlogger_conf(self):
+        f = open("tests/pmlogger.conf", "r")
+        pmlogger_conf = f.read()
+        input = pcp_plugin.PcpInputParams(
+            pmlogger_interval=1.0,
+            pmlogger_metrics="kernel.all.cpu.user mem.util.used",
+            timeout=3,
+            pmlogger_conf=pmlogger_conf,
+        )
+
+        output_id, output_data = pcp_plugin.StartPcpStep.start_pcp(
+            params=input, run_id="ci_pcp"
+        )
+
+        print(f"==>> output_id is {output_id}")
+        print(f"==>> output_data is {output_data}")
+
+        self.assertEqual("success", output_id)
+        self.assertIsInstance(output_data.pcp_output, list)
```

Comment:
Two questions:
1. Doesn't the assertion at line 166 print out the value of `output_id` if the assertion fails?...in which case there's little need to print it unconditionally.
2. The assertion at line 167 is pretty weak:  I presume that the output should be a list based on the plugin's schema, so this check doesn't really say much about whether the plugin actually ran.  Shouldn't we be testing the _contents_ of the list in some way?  (Shouldn't we at least check that the list is not empty??)

Ditto for `test_functional_user_pmrep_conf()`.

Reply:
The prints are a little extraneous here, but they've been helpful for quick basic debug while testing changes.

Reply:
The assertion there is definitely garbage. I'm going to update all of those here in the functional tests to test the serialization of the output against the schema.

Reply:
I think that that was probably a valuable update, but I don't think it actually improved the situation:  it could still be the case that the plugin returns valid but obviously incorrect data (e.g., an empty list), and the test will pass.

Reply:
True. Again, this plugin breaks the norms of our plugin philosophy by accepting an `Any` type in the output, so in this case the schema validation of the output isn't as useful as it would be in another plugin. Since we aren't defining in the output schema at all for this plugin except that we expect a list, I'm not sure what else I can do here that is much better.

Reply:
Presumably we expect the plugin to actually produce output...so, the test could assert that the list is not empty.